src/components/CommentList/CommentList.tsx

```diff
@@ -1,5 +1,30 @@
 import React from 'react';
+import { Comment } from '../../types/types';
+import { CommentInfo } from '../CommentInfo';
 
-export const CommentList: React.FC = () => (
-  <>Put the list here</>
-);
+type Props = {
+  list: Comment[];
+  idOfPost: number;
+};
+
+export const CommentList: React.FC<Props> = (props) => {
+  const array = props.list.find(
+    comment => comment.postId === props.idOfPost
+    && <CommentInfo comment={comment} key={comment.id} />,
+  );
```

Comment:
Store the component in a variable instead of inline, which improves code readability 😌.

Reply:
also, you might not need `&& <CommentInfo` here cause it always converts to `true` which is useless in the function